build_test/pubspec.yaml

```diff
@@ -16,13 +16,13 @@ dependencies:
   html: ">=0.9.0 <0.15.0"
```

Comment:
Do we want to allow 0.15 given https://github.com/dart-lang/html/pull/152/files ?

Reply:
🤷